verticapy/connection/connect.py

```diff
@@ -95,6 +110,25 @@ def set_connection(conn: Connection) -> None:
     ODBC connection. This should not be confused with a
     native   VerticaPy   connection  created   by   the
     'new_connection' function.
+
+    Example
+    -------
+    .. code-block:: python
```

Comment:
Can you separate the code into two blocks and bring the comment outside.

1. Let's use the official Vertica Python client.
.. note:: You can use any connectors you prefer (ODBC, JDBC, etc.), as long as they have both `fetchone` and `fetchall` methods. However, it's worth noting that VerticaPy works most efficiently with the native client due to its support for various complex data types and certain Vertica optimizations.

2. We can then set up the connector.
.. warning:: This connector is used throughout the entire API, so if it's closed, you'll need to create a new one.